README.md

```diff
@@ -29,25 +48,34 @@ addEventListener('fetch', event => {
   event.respondWith(handleEvent(event))
 })
 
-const customKeyModifier = url => {
-  //custom key mapping optional
-  if (url.endsWith('/')) url += 'index.html'
-  return url.replace('/docs', '').replace(/^\/+/, '')
-}
-
 async function handleEvent(event) {
   if (event.request.url.includes('/docs')) {
     try {
-      return await getAssetFromKV(event, { mapRequestToAsset: customKeyModifier })
+      return await getAssetFromKV(event)
```

Comment:
Remove modifying `mapRequestToAsset` to the first example since it's illustrated below